configstack/graphviz.go

```diff
@@ -0,0 +1,42 @@
+package configstack
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/terragrunt/options"
+	"io"
+	"path/filepath"
+	"strings"
+)
+
+// WriteDot is used to emit a GraphViz compatible definition
+// for a directed graph. It can be used to dump a .dot file.
+func WriteDot(w io.Writer, terragruntOptions *options.TerragruntOptions, modules []*TerraformModule) error {
```

Comment:
Since we already have terraform included as a library, any reason why we can't use https://github.com/hashicorp/terraform/tree/master/digraph?

Reply:
I think I could use but unfortunately in their implementation there's no way to set properties on the nodes like color which are useful to know in the terragrunt graph (e.g. to know which dependency is not going to be executed)

Reply:
Ah gotcha. Can you add a comment here with that message so that we don't ask the same question again a few months down the line? Thanks!